codex-rs/rmcp-client/src/oauth.rs

```diff
@@ -58,6 +58,26 @@ pub struct StoredOAuthTokens {
     pub token_response: WrappedOAuthTokenResponse,
 }
 
+/// Determine where Codex should store and read MCP credentials.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum OAuthCredentialsStoreMode {
+    /// Keyring when available, otherwise file.
+    /// Credentials stored in the keyring will only be readable by Codex unless the user explicitly grants access via OS-level keyring access.
+    Auto,
```

Comment:
Should this have `#[default]`? Then you can forgo the `impl`?

Reply:
Perfect, thanks!